library/common/config_template.cc

```diff
@@ -70,6 +70,19 @@ const char* stats_sink_template = R"(
           cluster_name: stats
 )";
 
+const std::string config_header = R"(
```

Comment:
I dont think I fully understand why this become the config header? And its relation to the config_template.

Reply:
Let me expand in the PR description.

Reply:
Updated.